opensrp-chw-core/src/test/java/org/smartregister/chw/core/model/CoreFamilyMemberModelTest.java

```diff
@@ -38,4 +40,34 @@ public void getFamilyBaseEntityIdTest() {
         CoreFamilyMemberModel coreFamilyMemberModel = new CoreFamilyMemberModel(lastName, baseEntityId, familyBaseEntityId, entityType);
         Assert.assertEquals(coreFamilyMemberModel.getFamilyBaseEntityId(), familyBaseEntityId);
     }
+
+    @Test
+    public void testBaseEntityId() {
```

Comment:
Kindly follow the test naming convention `test[MethodName][ExpectedResult][Parameters]`